Kubernetes/omsagent.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: kube-system
 ---
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
{{- if semverCompare ">=1.16.0" .Values.global.Versions.Kubernetes }}
    apiVersion: apps/v1
{{- else }}
    apiVersion: extensions/v1beta1
{{- end }}

Reply:
Please chedk the min k8s version version for non-AKS.